.hooks/pre-push

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+printf "\n⏳ composer lint" && \
+vendor/bin/php-cs-fixer fix --verbose --dry-run --show-progress=none && \
+printf "\n⏳ npm lint" && \
+npm run lint && \
+printf "\n⏳ composer analyse" && \
+vendor/bin/phpstan analyse --memory-limit 512M --no-progress && \
```

Comment:
why not run `compser lint` and `composer analyse` for these steps?

Reply:
The composer scripts run the `--ansi` variants which messes with some git GUIs' console output